pkg/cloudprovider/external.go

```diff
@@ -35,12 +36,27 @@ func IsCloudProviderExternal(platformStatus *configv1.PlatformStatus, featureGat
 		configv1.PowerVSPlatformType,
 		configv1.VSpherePlatformType:
 		return true, nil
+	case configv1.ExternalPlatformType:
+		return isExternalPlatformEnabled(infrastructure)
 	default:
 		// Platforms that do not have external cloud providers implemented
 		return false, nil
 	}
 }
 
+func isExternalPlatformEnabled(infrastructure *configv1.Infrastructure) (bool, error) {
```

Comment:
since this is also checking the ccm status perhaps we could name it `isExternalPlatformCCMEnabled`, wdyt?

Reply:
it makes sense, I'll update it :+1: